recipes/bxa/meta.yaml

```diff
@@ -0,0 +1,59 @@
+
+{% set name = "bxa" %}
+{% set version = "3.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa81715c0f4f2a0cea84b802f3b3ea8154f153d547ce24b8c030d8442934cd82
+
+build:
+  number: 0
+  # may need to later add "skip: True  # [win]" to disable Windows.
+  skip: true  # [win or py<30]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
Is this really required? As you are not compiling code, `numpy`  can probably be omitted.